kmod/src/inode.c

```diff
@@ -1541,8 +1542,6 @@ void scoutfs_evict_inode(struct inode *inode)
 
```

Comment:
As we remove the inode destruction from evict_inode, we should remove the comment that talks about removing the persistent items.